beetsplug/beatport.py

```diff
@@ -261,10 +263,10 @@ def __init__(self, data):
             self.musical_key = six.text_type(data['key'].get('shortName'))
 
         # Use 'subgenre' and if not present, 'genre' as a fallback.
-        if 'subGenres' in data:
+        if 'subGenres' in data and data['subGenres']:
```

Comment:
Sometimes I use `if data.get('subGenres')` as a shorthand way of checking for both presence and non-emptiness.

Reply:
Ha, did not know I could do it this way.